Clarify the scope of this PER in the meta document (#63)
php-fig/per-coding-style: meta.md

> of guidelines to be used among all those projects. Thus, the benefit of this guide is
> not in the rules themselves, but in the sharing of those rules.

This PER extends, expands and replaces PSR-12, and is therefore also an extension of PSR-1.
With PSR-12 being the base of this PER, a list of differences is provided below to assist with migration,
but it should be considered as an independent specification.

> It is a long term goal of this PER to incorporate coding syle requirements found in PSR-1,

Reviewer: ```suggestion
It is a long term goal of this PER to incorporate coding style requirements found in PSR-1,
```

> It is a long term goal of this PER to incorporate coding syle requirements found in PSR-1,
> so that all coding style guidelines can eventually be found in one specification.
> However, this PER is not a replacement for PSR-1 and requires adherence to it and any errata.

Reviewer 2: ```suggestion
This PER will eventually incorporate and replace any stylistic requirements found in PSR-1.
PER-1 includes a number of non-stylistic requirements that affect runtime behavior, which are
out of scope for this PER.

If a particular rule is mentioned in both this PER and PSR-1, this PER takes precedence. If not,
the PSR-1 rule (including errata) still applies.
```

This PER will include coding style guidelines related to new functionality added to PHP
after the publication of PSR-12, as well as clarifications on the text of PSR-12.
It is a goal of this PER to maintain style consistency between related old and new PHP syntax,
such as class constants and constants in enumerations. In rare cases, this may cause new
releases to include coding style guidelines that break backwards compatibility, limited by
the non-goals and versioning sections below.

### 3.2. Non-Goals

> It is not the intention of this PER to substantially or arbitrarily alter coding style guidelines found in PSR-1 and PSR-12.
> It will not change anything stipulated in PSR-1 that concerns technical interoperability or runtime behaviour.

Reviewer 2: ```suggestion
* It is not the intention of this PER to arbitrarily alter coding style guidelines found in PSR-1 and PSR-12.
* It is not the intention of this PER to change any elements of PSR-1 that concern technical interoperability or runtime behavior.
```

Author: I perceive this suggestion as less firm on not changing runtime elements of PSR-1 and not making large changes (measured by number of rules touched), only a single issue we've identified as possibly in scope. I'm concerned that may make it less pallatable to anyone on the fence about this change. Wdyt?
